Formula/heroku.rb

```diff
@@ -1,9 +1,9 @@
 class Heroku < Formula
   desc "Everything you need to get started with Heroku"
   homepage "https://cli.heroku.com"
-  url "https://cli-assets.heroku.com/branches/stable/5.9.0-1b8deac/heroku-v5.9.0-1b8deac-darwin-amd64.tar.xz"
-  version "5.9.0"
-  sha256 "37a578261bad805788a24acbb76cf70adef258b5ee4d75894af3691dcc73f02e"
+  url "https://cli-assets.heroku.com/heroku-cli/channels/stable/heroku-cli-v6.6.7-ecf0119-darwin-x64.tar.xz"
+  version "6.6.7-ecf0119"
```

Comment:
Any reason not to make it just "6.6.7" as the tag is named https://github.com/heroku/cli/releases?